packages/node_modules/@webex/plugin-memberships/test/integration/spec/memberships.js

```diff
@@ -606,18 +606,35 @@ describe('plugin-memberships', function () {
       });
     });
 
-    // TODO add a test for hiding a one on one space when that
-    // feature goes GA  (see the one-on-one test in plugin-rooms)
-    /*
     describe('#hide()', () => {
-      let membership, room;
+      let membership, roomId;
 
       // We need a one-on-on space for this test
       // We create it by sending a message to the test user
-      before(() => {
+      before(() =>
         // Ensure that room is a one-on-one object
-        // Ensure that membership is the SDK users membership object
-      });
+        webex.messages.create({
+          toPersonId: user1.id,
+          text: 'This message will create a 1-1 space'
+        })
+          .then((message) => {
+            roomId = message.roomId;
+
+            // Get the test users membership
+            return webex.memberships.list({
+              roomId,
+              personId: user.id
+            });
+          })
+          .then((memberships) => {
+            // Ensure that membership is the SDK users membership object
+            membership = memberships.items[0];
+            validateMembership(membership);
+            assert((membership.personId === actor.id),
+              '#hide test did not find test users membership to hide');
+          })
+          .catch((e) => console.error(e.message)));
```

Comment:
We should rethrow the error so the test fails during the setup phase.

Reply:
Removed the catch from the setup logic.